docs/resources/cluster.md

```diff
@@ -53,7 +53,7 @@ A resource for managing clusters on Instaclustr Managed Platform. A cluster cont
 
 | Property             | Description                                                    | Default  |
 |----------------------|----------------------------------------------------------------|----------|
-| `iam_principal_arns` | List of IAM Principal ARNs to add to the endpoint service      | Required |
+| `iam_principal_arns` | List of IAM Principal ARNs to add to the endpoint service. This field is updatable      | Required |
```

Comment:
I think it's not **Required** anymore after Vu's api changes? But if someone only has an empty `privateLink{}` block, Terraform won't read this block in, and therefore that CDC won't set PrivateLink enabled. So it's more like **Required, but can be an empty list**.